app/controllers/v1/forams_controller.rb

```diff
@@ -1,7 +1,7 @@
 module V1
   class ForamsController < ApplicationController
     before_action :set_foram, only: :show
-
+    before_action :authenticate_user!
```

Comment:
tak jak wspominałem kiedyś. Nie wiem czy będzie potrzebne zabezpieczenie kontrolerów. Póki co wstawiłem to do testów. Przed mergem to wyrzucę